app/helpers/wizards/steps/signup/sektion/various_fields_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module Wizards::Steps::Signup::Sektion::VariousFieldsHelper
+  def entry_date_text
```

Comment:
solange wir das nur innerhalb von einem Step anzeigen würde ich dafür keine neuen Helper machen, sondern die Methode in den step integrieren. 

Zudem sollte die Infos, nicht hard codiert sondern von `SacMembershipConfig` gelesen werden und die Übersetzungen mit Datum und Prozent parametriert werden, sonst stimmt der text nicht mehr, falls sie dort mal die werte anpassen